### PR TITLE
feat(FOROME-1226): added Tooltip component for UI kit

### PR DESCRIPTION
**Pull request by AlMaQntr**

### CODE REVIEW

**Comment by QSdmitrioul** [comment]
You revert changes on purpose?

**Review by QSdmitrioul — CHANGES_REQUESTED** [review]
You revert changes on purpose?

**Review by QSdmitrioul — APPROVED** [review]

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/ui/icon/icon.tsx [hunk 1]
```diff
@@ -23,61 +23,70 @@ export interface IIconProps {
...
-  stroke,
-  fill,
-  className,
-  onClick,
-  onMouseDown,
-  onMouseUp,
-  dataTestId,
-}: IIconProps): ReactElement => {
-  const icon: IconItem = icons[name]
+export const Icon = forwardRef<SVGSVGElement, IIconProps>(
```
Comment (AndreyUstyumenko) [inline src/ui/icon/icon.tsx:26]: why forwardRef?

Comment (AlMaQntr) [reply src/ui/icon/icon.tsx:26]: because it's need to pass ref from Tooltip component

**Pull request merged**